pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.sagebionetworks.bridge</groupId>
     <artifactId>sdk-group</artifactId>
     <!-- Please use the maven versions plugin to manage this, e.g.`mvn versions:set -DnewVersion=0.8.1`-->
-    <version>0.12.23</version>
+    <version>0.12.24</version>
```

Comment:
Looks like Dwayne took 24 in develop, I just updated my PR to 26, you should be able to take 25